### DISCUSSION TITLE
Is the issues tab hidden by design?

### DISCUSSION THREAD

**Discussion by creutz** [spatie/laravel-sitemap#467] [General]
I'd like to post an issue but it seems its not possible here for this repo. I believe the maintainer has to allow it. Whats the preferred way to post a bug?

Best,
Alex

**Comment by freekmurze** [comment]
The preferred way is you creating a PR to fix it 🙂 

Second best: create a discussion describing the bug, and when we, of somebody else from the community has time, it will be fixed.

**Reply by creutz** [reply]
Thanks. I didnt look into the issue too much, hence I have no fix at hand. Maybe its not even a bug. I want to add a URL like this:

```
<?php
class GenerateSitemap extends Command
{
    protected $signature = 'sitemap:generate';
    public function handle()
    {
        $sitemap = Sitemap::create();
        $url = Url::create('/de/afolderwithstuff/');
        $url->setChangeFrequency(Url::CHANGE_FREQUENCY_YEARLY)->setPriority(0.5);
        $sitemap->add($url);
        $sitemap->writeToFile($path);
    }
}
```
But in the final sitemap the trailing slash of the url is being removed, it appears as `<loc>mydomain.com/de/afolderwithstuff</loc>`. Any idea what I am doing wrong?

**Reply by ssnepenthe** [reply]
I know this is an old discussion but... The sitemap views use the `url()` helper which calls `UrlGenerator::to()` which trims the trailing slash:

https://github.com/illuminate/routing/blob/49eed1209f0dbce87cc1338ba67068d38a1bf813/UrlGenerator.php#L208-L231

That method returns the URL unmodified if it is already a valid absolute URL, so the simple workaround is to provide the full URL instead of just the path. e.g.:

```php
$url = Url::create(config('app.url') . '/de/afolderwithstuff/');
```
